src/ProgressBar/ProgressBar.tsx

```diff
@@ -45,7 +57,6 @@ const ProgressBar = ({ percentage, iconEndName, showPercentage, ...props }) => (
     {iconEndName && (
       <Icon
         ml="smaller"
-        flex="0"
```

Comment:
`Icon` also doesn't support `flex`, and this also doesn't appear to hurt anything.